index.php

```diff
@@ -8,12 +8,13 @@
 
 $PDO = connectDb($host, $username, $password, $dbName);
 
-if(count($_POST)>0){
-    addBottle($PDO, $_POST['itemname'], $_POST['purchaselocation'], $_POST['type'], $_POST['purchasedate']);
+$errorMessage = "";
+if(count($_POST)>0) {
+    if (checkFormSubmission($_POST)[0]) {
+        addBottle($PDO, $_POST['item-name'], $_POST['purchase-location'], $_POST['type'], $_POST['purchase-date']);
+    } else $errorMessage = checkFormSubmission($_POST)[1];
```

Comment:
a couple things here:
- it is better to only call the function `checkFormSubmission` once and use the results, the way you have it it runs the function twice, so it would be more efficient to only run it once
- you've decided that the 0th item in the returned array is implicitly the result from the validation (`true` or `false`) and the 1st item is implicitly the error message or "".  it's better to explicitly name them, so the code reads more intuitively e.g. `validationResult['message']` rather than `validationResult[1]`